toml/src/main/kotlin/org/rust/toml/resolve/CargoTomlFileReferenceProvider.kt

```diff
@@ -39,7 +42,15 @@ class CargoTomlFileReferenceProvider(private val patternType: PathPatternType) :
         }
 
         if (kind !is TomlLiteralKind.String) return emptyArray()
-        return CargoTomlFileReferenceSet(element, completeDirs, completeRustFiles).allReferences
+        val references = CargoTomlFileReferenceSet(element, completeDirs, completeRustFiles).allReferences
+
+        val ignoreGlobs = patternType == WORKSPACE &&
+            element.parentOfType<TomlKeyValue>()?.key?.segments?.firstOrNull()?.name == "members"
```

Comment:
Why do we check only `members` ? As I see, glob patterns can be used in all arrays in `workspace` table

Reply:
The documentation states that it works only in members I think. I tried globs in exclude and they do not seem to work (i.e. exclude seems to consider * to be a normal path character).

Reply:
Hm, interesting
Looks like it doesn't work in `exclude` (but I actually was sure that it should) but it works in `default-members` section

For example, for the following project

<img width="388" alt="Screen Shot 2021-06-28 at 19 18 19" src="https://user-images.githubusercontent.com/2539310/123670186-9d664380-d845-11eb-8a92-581a24339608.png">

with root Cargo.toml
```toml
[workspace]
members = [
    "lesson1/*/",
    "lesson2/*/",
]
default-members = [ "lesson1/*/" ]
```

`cargo build` compiles only `lesson1/task1` and `lesson1/task3`

Reply:
Added `default-members` to the supported key list.